restcomm/restcomm.application/src/main/webapp/WEB-INF/scripts/mariadb/init.sql

```diff
@@ -188,7 +188,8 @@ voice_method VARCHAR(4),
 voice_fallback_url MEDIUMTEXT,
 voice_fallback_method VARCHAR(4),
 voice_application_sid VARCHAR(34),
-uri MEDIUMTEXT NOT NULL
+uri MEDIUMTEXT NOT NULL,
+push_client_identity VARCHAR(34)
```

Comment:
@agafox I believe this is not the proper approach and this approach will not be enough to cover all use cases. Keep in mind I don't have 100% clear picture of the feature.

Let me elaborate on my thoughts so we can discuss here.

Let's say we have client `bob`.

Bob registers from 2 sip clients at the same time:
1. Cisco desk phone (or any other softphone on laptop etc that doesn't need push notification)
2. Android Olympus application (that needs push notification)

Because Restcomm keeps two active registrations for client `bob`, whenever Restcomm needs to reach to `bob` it will Dial to the two AOR according to the registrations table.

As long as the mobile device is active and Android Olympus responds to Option requests from Restcomm, the dial fork will work fine.

When the mobile device goes to sleep and no longer responds to Option requests, Restcomm will remove the Android registration and will keep only the registration for the Cisco desk phone. So when someone calls `bob`, only the Cisco phone will get INVITE.

Even if CallManager sees that this client needs to send a Push Notification (checking the new table column), the AOR in the DB will be of the Cisco device only and will not have the Android AOR.

Even if we don't remove the mobile client's Registration on timeout (bad idea - just for example) and we just provide the Client SID in the DB column, or any other flag indicating that the client needs Push Notification, Restcomm won't be able to tell which of the 2 registrations needs Push Notification and will send notification to both Cisco and Android.

Modifying the Client model to support Push Notification is bad idea because in order to work properly we will have to give up the Dial Forking that now we support.

Currently, Dial Bob will dial to ANY AOR in the DB for Bob.

With the extra property indicating the need for Push Notification, we need different client for `bob_deskphone`, and another one for mobile `bob_android`.

I believe the proper way will be to implement this feature at the Registrar, which should check the User-Agent header and if the sip client is a mobile application should store the AOR for the Push Notification service. We should have a new table for this.